src/task-2/index.ts

```diff
@@ -0,0 +1,33 @@
+import csv from "csvtojson";
+import { createReadStream, createWriteStream, existsSync, mkdirSync } from "fs";
+import path from "path";
+import stream from "stream";
+import { promisify } from "util";
+
+const pipeline = promisify(stream.pipeline);
+
+const initialMemoryUsed = process.memoryUsage().heapUsed / 1024 / 1024;
+
+const inputPath = path.join(__dirname, "csv/test.csv");
+const outputDir = path.join(__dirname, "json");
+const outputPath = path.join(outputDir, `${new Date().getTime()}.txt`);
+
+if (!existsSync(outputDir)) {
+  mkdirSync(outputDir);
+}
+
+const errorHandler = (error: Error) => console.error.bind(console, error);
```

Comment:
why do you need to do this? You don't have this pointing inside, so nothing to bind

Reply:
Removed bind